pkg/controller/virtualservice/util/util.go

```diff
@@ -36,7 +36,6 @@ const (
 var ApplicationLabels = [...]string{
 	ApplicationNameLabel,
 	ApplicationVersionLabel,
-	AppLabel,
```

Comment:
`app` label is required by istio, why do we remove it?

Reply:
Use another AppLabels , which only have the necessary labels for kubernetes-sigs/application. So the resources ,such as ingress which has no `app` labels can be treated as application.